Plugins/Flow.Launcher.Plugin.Program/Programs/Win32.cs

```diff
@@ -83,6 +83,7 @@ private static MatchResult Match(string query, IReadOnlyCollection<string> candi
         public Result Result(string query, IPublicAPI api)
         {
             string title;
+            string titleOnly;
```

Comment:
I think this variable name doesn't provide any information. and is this variable needed? In either case the value is the same as `resultName`.

Reply:
Yeah, no need for an extra variable. I just noticed that. We can use `resultName` as `AutoCompleteText`.